.github/workflows/platform.updateReadMe.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         shell: pwsh
         run: |
           # Load used functions
-          . "$env:GITHUB_WORKSPACE/utilities/tools/Set-ReadMeModuleTable.ps1"
+          . (Join-Path "$env:GITHUB_WORKSPACE 'utilities' 'tools' 'platform' 'Set-ReadMeModuleTable.ps1')
```

Comment:
Should the script path 'utilities/tools/platform' be an environment variable?

Reply:
See comment above